examples/document_loading/csv/index.ts

```diff
@@ -0,0 +1,91 @@
+import Instructor from "@/instructor"
+import OpenAI from "openai"
+import { z } from "zod"
+import { CSVLoader } from "langchain/document_loaders/fs/csv";
+
+// Create a CSV loader instance
+const loader = new CSVLoader("./examples/document_loading/csv/transactions.csv")
+
+// Load the CSV document
+const doc = await loader.load()
```

Comment:
why would loading the civ via langchain be any better then just reading it raw into a string - since ur jsut passing it as a string back to the completion?

eg. 

  const fileContent = fs.readFileSync(csvFilePath, { encoding: 'utf-8' });
  
  vs 
  
  the loader + converting to json with the langchaing document types etc jsut to turn back to a string?

Reply:
are there other common use cases with langchain document tools that one would benefit from using?

Reply:
ya more so as a very simple example to begin the convo. Once loaded, especially with longer docs, you would want to split the text and index. This was really a response to the loaders available per our twitter convo about Maybe

Reply:
Wait, why do we need Lang chain to load a CSV file?